zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateProcessInstanceTest.java

```diff
@@ -338,12 +338,12 @@ private CreateProcessInstanceCommandStep1 getCommand(
     return useRest ? createInstanceCommand.useRest() : createInstanceCommand.useGrpc();
   }
 
-  private void deployProcesses(final TestInfo testInfo) {
+  private void deployProcesses(final TestInfo testInfo, final boolean useRest) {
```

Comment:
🔧 Neat trick but I think we need to adjust the `CreateDeployment...Test` classes to thoroughly check all deployment use cases work equally well.

Reply:
I missed those tests 🤔